docs/technical_reference/unit_models/zero_order_unit_models/landfill_zo.rst

```diff
@@ -16,6 +16,23 @@ Costing Method
 Costing is calculated using the cost_landfill method in the zero-order costing package.
 See documentation for the :ref:`zero-order costing package<zero_order_costing>`.
 
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "capacity basis for capital cost", "capacity_basis", "kg/hr"
+   "total mass flow rate", "total_mass", "kg/hr"
+
+Additional Constraints
+----------------------
+
+.. csv-table::
+   :header: "Description", "Constraint Name"
+
+   "Total mass constraint", "total_mass_constraint"
```

Comment:
Another passing comment - I believe there are tools in IDAES that can render Pyomo expressions as LaTeX which might allow us to include the actual expression in the docs if we want to.

Reply:
Can you point me to where those tools are?

Reply:
I think they might be here: https://github.com/IDAES/idaes-pse/blob/main/idaes/core/util/expr_doc.py

However, the best person to ask is @eslickj as these are some of his useful-but-no-so-well-documented tools.

Reply:
I had a look and (very quickly) tried using `document_constraints` and I got back some error that I was unsure of how to deal with. I'd be happy to discuss with [@eslickj](https://github.com/eslickj) in the future, but I think we'd better keep this for a subsequent PR.